packages/connect-migrate/src/migrations/v2.0.0.ts

```diff
@@ -119,7 +118,7 @@ const bufGenYamlMigrations: BufGenYamlMigration[] = [
     updatePlugin: {
       remote: "buf.build/connectrpc/query-es",
       from: "^1.0.0",
-      to: "1.4.2", // TODO
+      to: targetVersionConnectQuery,
```

Comment:
This will update the remote plugin to the beta version, which will not be available as a pre-release (only stable versions are). I think that's acceptable, since the migration is also beta.